Dangerfile

```diff
@@ -61,5 +61,4 @@ begin
 
 rescue => ex
   fail "There was an error with Danger bot's Junit parsing: #{ex.message}"
-  puts ex.inspect # view the entire error output in the log
 end
```

Comment:
It didn't like that line? I had that in there so we can read anything that goes wrong ... Is there another way to be able to read the output?

Reply:
yeah, it says that puts statement should not be included in dangerfile.
Even, that doesn't solve the problem. Now it says:-
`Your Dangerfile has had smart quotes sanitised. To avoid issues in the future, you should not use TextEdit for editing it. If you are not using TextEdit, you should turn off smart quotes in your editor of choice.`

Reply:
Should we ignore this error for now and let's put the puts line back??